org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corext/refactoring/changes/ChangeUtil.java

```diff
@@ -0,0 +1,267 @@
+/*******************************************************************************
```

Comment:
Not sure this class should be in this package. Why is it not in org.eclipse.jdt.ls.core.internal? It is glue code between LSP and jdt.coreext.

Reply:
agree, moved to `org.eclipse.jdt.ls.core.internal`